src/util/cnames.js

```diff
@@ -76,6 +76,9 @@ const parseCNAMEsFile = (content, context = {}) => {
         // Convert vercel.app to cname.vercel-dns.com
         if (target.endsWith('.vercel.app')) target = 'cname.vercel-dns.com';
 
+        // Convert netlify.com to netlify.app
+        if (target.endsWith('.netlify.com')) target = target.replace('.netlify.com', '.netlify.app');
```

Comment:
```suggestion
        if (target.endsWith('.netlify.com')) target = target.replace(/\.netlify\.com$/, '.netlify.app');
```
Just to avoid an edge case where `netlify.com` exists elsewhere in the string, we can use regex to replace the occurrence specifically at the end of the string

Reply:
> Just to avoid an edge case where `netlify.com` exists elsewhere in the string, we can use regex to replace the occurrence specifically at the end of the string

Adopted